akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -1324,6 +1328,131 @@ private[stream] object Collect {
     }
 }
 
+object AskStage {
+  final class AskStageTargetActorTerminatedException(ref: ActorRef)
```

Comment:
looks like this is exposed in public, in failStage, so should be defined elsewhere

Reply:
moved to akka.stream